### PR TITLE
Component tests

### PR DESCRIPTION
**Pull request by lisamso**

### CODE REVIEW

File: web/jasmine.config.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// Karma configuration
```
Comment (mariekt) [inline web/jasmine.config.js]: Er denne filen autogenerert? Var sinnsykt mange kommentarer, hehe 😬

**Comment by mariekt** [comment]
Jeg får dessverre en feilmelding i `MapComponent should create` 😞 

`Failed: Template parse errors:
Can't bind to 'queryParams' since it isn't a known property of 'a'. ("
    <a
      routerLink="/recipes"
      [ERROR ->][queryParams]="{term: search.term, category: search.category, continent: search.continent}"
      mat"): ng:///DynamicTestModule/MapComponent.html@5:6`

**Comment by AlpakkAn** [comment]
Feilen i MapComponent er fikset i map-tests branchen :D

**Comment by AlpakkAn** [comment]
Jeg får veldig ofte feilmeldingen

`Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.`

**Comment by AlpakkAn** [comment]
Får veldig ofte feilmeldingen

`Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.`

**Comment by Tirilsa** [comment]
Jeg får fortsatt samme feilen som An flere steder

**Pull request merged**